src/Starcounter.VS.CSApplicationProjectTemplate/ApplicationProjectTemplate.csproj

```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>$safeprojectname$</RootNamespace>
     <AssemblyName>$safeprojectname$</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v$targetframeworkversion$</TargetFrameworkVersion>
```

Comment:
@Mihaiii was this tested? Are you sure that `v` is not included into the `$targetframeworkversion$` variable value?

Reply:
Yes, it was tested with and without the "v". "v" is not included into the `$targetframeworkversion$` variable value.

Reply:
Okay, thanks for the clarification. I also add Urban to review, but it's approved from me.